be/src/exprs/expr_context.cpp

```diff
@@ -79,10 +80,10 @@ Status ExprContext::open(std::vector<ExprContext*> evals, RuntimeState* state) {
 }
 
 void ExprContext::close(RuntimeState* state) {
-    if (_closed) {
+    bool expected = false;
+    if (!_closed.compare_exchange_strong(expected, true)) {
```

Comment:
Why Exprcontext::close is called concurrently

Reply:
Previous multiple ScanOperator could close it concurrent.